dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCCloseWrappedResponseOutputAfterForwardTest.java

```diff
@@ -0,0 +1,124 @@
+/*******************************************************************************
```

Comment:
My understanding of this test is you're only testing the `true` scenario for all servlet versions. Can we also test the default behavior for servlet 5.0 and default value for previous servlet versions?